ops/model.py

```diff
@@ -42,6 +43,22 @@ def get_relation(self, relation_name, relation_id=None):
         """
         return self.relations._get_unique(relation_name, relation_id)
 
+    def get_binding(self, binding_key):
+        """Get a network space binding.
+
+        Providing a Relation instance is preferred because network info exposed in a Binding instance
+        can be different depending on whether a relation is a cross-model relation or not.
```

Comment:
This seems a bit confusing. If a space is bound to a relation, it shouldn't matter whether this relation is cross-model or not. What this should really be saying is whether the space is bound to an individual established relation or to all of the relations established against that endpoint, I suspect?